src/foam/core/holders.js

```diff
@@ -28,6 +28,22 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.core',
+  name: 'IdentifiedStringHolder',
+
```

Comment:
@KernelDeimos put the PR notes in here as documentation.

Reply:
Added - made it slightly more generic